pom.xml

```diff
@@ -17,418 +17,621 @@ limitations under the License.
-                <relocation>
-                  <pattern>org.apache.tinkerpop</pattern>
-                  <shadedPattern>cosmosdb_connector_shaded.org.apache.tinkerpop</shadedPattern>
-                </relocation>
-                <relocation>
-                  <pattern>org.apache.http</pattern>
-                  <shadedPattern>cosmosdb_connector_shaded.org.apache.http</shadedPattern>
-                </relocation>
-                <relocation>
-                  <pattern>com.google</pattern>
-                  <shadedPattern>cosmosdb_connector_shaded.com.google</shadedPattern>
-                </relocation>
-                <relocation>
-                  <pattern>com.google.common</pattern>
-                  <shadedPattern>cosmosdb_connector_shaded.com.google.common</shadedPattern>
-                </relocation>
-              </relocations>
-              <filters>
-                <filter>
-                  <artifact>*:*</artifact>
-                  <excludes>
-                    <exclude>META-INF/services/javax.annotation.processing.Processor</exclude>
-                    <exclude>META-INF/*.MF</exclude>
-                    <exclude>META-INF/*.SF</exclude>
-                    <exclude>META-INF/*.DSA</exclude>
-                    <exclude>META-INF/*.RSA</exclude>
-                  </excludes>
-                </filter>
-                <filter>
-                  <artifact>commons-logging:commons-logging</artifact>
-                  <includes>
-                    <include>**</include>
-                  </includes>
-                </filter>
-              </filters>
-              <artifactSet>
-                <excludes>
-                  <exclude>org.tachyonproject:tachyon-client</exclude>
-                  <exclude>org.apache.hadoop:*</exclude>
-                  <exclude>org.apache.spark:*</exclude>
-                  <exclude>org.scala-lang:*</exclude>
-                  <exclude>org.apache.tinkerpop:*</exclude>
-                </excludes>
-              </artifactSet>
-              <minimizeJar>true</minimizeJar>
-              <finalName>${project.artifactId}-${project.version}-uber</finalName>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-    </plugins>
-  </build>
-  <developers>
-    <developer>
-      <name>CosmosDB Programmability Devs</name>
-      <email>askcosmosdb@microsoft.com</email>
-      <organization>Microsoft</organization>
-      <organizationUrl>http://www.microsoft.com/</organizationUrl>
-    </developer>
-  </developers>
-  <scm>
-    <connection>scm:git:git@github.com:Azure/azure-cosmosdb-spark.git</connection>
-    <developerConnection>scm:git:git@github.com:Azure/azure-cosmosdb-spark.git</developerConnection>
-    <url>git@github.com:Azure/azure-cosmosdb-spark.git</url>
-  </scm>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-cosmosdb-spark_2.2.0_2.11</artifactId>
```

Comment:
For master, should we focus on Spark 2.2.1  (matching default Databricks Runtime 3.5) or Spark 2.2.0 (HDI 3.7, Databricks Runtime 3.4).  Note, that we should have a Spark 2.3 version  of this in the Spark 2.3 branch  (matching Databricks Runtime 4.0).